### DISCUSSION TITLE
How to Increase Max Attachment Size for SMTP (not API)?

### DISCUSSION THREAD

**Discussion by izzoa** [postalserver/postal#2341] [Help with using Postal]
Hi - did a good search through Github issues as well as discussions and didn't see anything up-to-date on this topic (but apologies if I missed something!); is it possible to increase the max file attachment size for emails that are sent via SMTP (not API)?

**Comment by willpower232** [comment]
What size are you trying to send? Are you encountering a problem at the minute?

**Reply by izzoa** [reply]
Nothing terribly large, 22MB fails to send currently.

**Reply by willpower232** [reply]
Well that is a bit big for an email IMHO but you probably need to add the config value with a higher number to your config file

https://github.com/postalserver/postal/blob/9f4ef8f57a839c5529b4f00a36b832740386b4ed/config/postal.defaults.yml#L76

You may also need to tweak your MySQL settings like this https://github.com/postalserver/postal/issues/324#issuecomment-323538575
